src/main/java/pro/belbix/ethparser/dto/DeployerDTO.java

```diff
@@ -0,0 +1,39 @@
+package pro.belbix.ethparser.dto;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.Instant;
+
+
+@Entity
+@Table(name = "deployer_tx", indexes = {
+        @Index(name = "idx_deployer_tx", columnList = "block")
+})
+@Cacheable(false)
+@Data
+public class DeployerDTO implements DtoI
+{
```

Comment:
please, follow the Java convention and open bracers on the same line with class declaration
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
Class and Interface Declarations
you can use google adopted version and integrate a config to IDE
as example IntelliJ 
https://stackoverflow.com/questions/42979700/how-to-configure-google-java-code-formatter-in-intellij-idea-17